test/fixtures/dialects/postgres/postgres_drop_procedure.sql

```diff
@@ -0,0 +1,13 @@
+DROP PROCEDURE do_db_maintenance();
+
+drop procedure insert_actor;
+
+drop procedure insert_actor(varchar);
+
+drop procedure insert_actor(varchar, varchar);
+
+drop procedure delete_actor, update_actor;
+
+drop procedure delete_actor, update_actor CASCADE;
+
+drop procedure delete_actor(in id varchar);
```

Comment:
Would be good to add a test case with two types of commas, as SQLFluff sometimes struggles with this:

```suggestion
drop procedure delete_actor(in id varchar);

drop procedure insert_actor(varchar, varchar), insert_actor2(varchar, varchar);
```

I've tested and it works fine so let's add a test case so it stays that way!

Reply:
On it! :+1: